Raven.Database/Indexing/QueryBuilder.cs

```diff
@@ -52,53 +53,56 @@ public static Query BuildQuery(string query, RavenPerFieldAnalyzerWrapper analyz
 
         public static Query BuildQuery(string query, IndexQuery indexQuery, RavenPerFieldAnalyzerWrapper analyzer)
         {
-            if (UseLuceneASTParser)
+            using (new TemporaryCulture(CultureInfo.InvariantCulture))
```

Comment:
@aviviadi We have something similar already
https://github.com/ayende/ravendb/blob/v3.5/Raven.Database/Util/CultureHelper.cs